toolboxTreppe/toolboxTreppe.ino

```diff
@@ -130,5 +128,4 @@ const uint16_t XY(const uint8_t& x, const uint8_t& y) {
     returnValue += x;
   }
   return returnValue;
-}
-
+}
```

Comment:
there should be an empty line at the end

Reply:
already changed on thursday but not pushed yet